card/lib/cardio.rb

```diff
@@ -11,27 +11,11 @@ module Cardio
     Card::Loader.load_mods if Card.count > 0
   end
 
-  class << self
-    def paths
-      @@paths
-    end
-
-    def config
-      @@config
-    end
+  mattr_reader :paths, :config, :root, :cache
 
-    def root
-      @@root
-    end
-
-    def gem_root
-      CARD_GEM_ROOT
-    end
-
-    def add_card_paths paths, config, root
-      @@config = config
-      @@paths = paths
-      @@root = root
+  class << self
+    def card_config *args
+      @@config, @@paths, @@root, @@cache = args
```

Comment:
If we can figure out a way to bind the different config domains together, maybe we wouldn't need this.  This at least simplifies and puts it in one place.